MdeModulePkg/Bus/Usb/UsbMouseAbsolutePointerDxe/UsbMouseAbsolutePointer.c

```diff
@@ -160,7 +160,14 @@ USBMouseAbsolutePointerDriverBindingStart (
   }
 
   UsbMouseAbsolutePointerDevice = AllocateZeroPool (sizeof (USB_MOUSE_ABSOLUTE_POINTER_DEV));
-  ASSERT (UsbMouseAbsolutePointerDevice != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (UsbMouseAbsolutePointerDevice == NULL) {
+    ASSERT (UsbMouseAbsolutePointerDevice != NULL);
+    Status = EFI_OUT_OF_RESOURCES;
+    goto ErrorExit1;

```

Comment:
Since the call to `OpenProtocol()` succeeded, this error path should close it. Since `ErrorExit` already contains a `UsbMouseAbsolutePointerDevice` null check and since you're setting `Status = UsbMouseAbsolutePointerDevice`, you could use it.

Reply:
Updated